apps/api/src/service/applicationService.ts

```diff
@@ -28,15 +28,30 @@ import { failure, success, type AsyncResult } from '@/utils/results.js';
 import { ApplicationStates, ApplicationStateValues } from '@pcgl-daco/data-model/src/types.js';
 import { applicationActionSvc } from './applicationActionService.js';
 import {
+	type ApplicationContentModel,
 	type ApplicationContentUpdates,
+	type ApplicationRecord,
 	type ApplicationsColumnName,
+	type ApplicationStateTotals,
 	type ApplicationUpdates,
 	type JoinedApplicationRecord,
 	type OrderBy,
 } from './types.js';
 
+/**
+ * ApplicationService provides methods for Applications DB access
+ * @param db - Drizzle Postgres DB Instance
+ * @method createApplication: Create new Application record
+ * @method editApplication: Update Application Contents and parent Application record
+ * @method findOneAndUpdate: Update a base Application record
+ * @method getApplicationById: Find a specific Application record
+ * @method getApplicationWithContents: Find an Application record with Contents included
+ * @method listApplications: Find multiple sorted Application records
+ * @method applicationStateTotals: Obtain count for all Application records with each State
+ */
+
 const applicationSvc = (db: PostgresDb) => ({
-	createApplication: async ({ user_id }: { user_id: string }) => {
+	createApplication: async ({ user_id }: { user_id: string }): AsyncResult<ApplicationRecord> => {
```

Comment:
In this function, we call `await actionRepo.create(newApplicationRecord[0]);`, and we are planning to remove this action from the application action history. This is just a reminder that we will remove that from this function.